pom.xml

```diff
@@ -347,7 +347,8 @@
         <version.org.cryptacular>1.2.4</version.org.cryptacular>
         <version.org.eclipse.jdt.core.compiler>4.6.1</version.org.eclipse.jdt.core.compiler>
         <version.org.eclipse.microprofile.config.api>1.4</version.org.eclipse.microprofile.config.api>
-        <version.org.eclipse.microprofile.fault-tolerance.api>2.1.1-RC1</version.org.eclipse.microprofile.fault-tolerance.api>
+        <version.org.eclipse.microprofile.fault-tolerance.api>2.1</version.org.eclipse.microprofile.fault-tolerance.api>
```

Comment:
@rhusar I guess this has the same contents as the 2.1.1-RC1 version?

Reply:
@kabir Yes, the API is the same, there are some javadoc improvements in later versions, but I don't want to risk MP FT doesn't release a final before we do and to cause confusion. So splitting the API and TCK modules – which in all honesty should be versioned separately but oh well – gives us flexibility.